src/GraphRequest.ts

```diff
@@ -302,6 +303,29 @@ export class GraphRequest {
 		}
 	}
 
+	/**
+	 * @private
+	 * Checks if the content-type is present in the _headers property. If not present, defaults the content-type to application/json
```

Comment:
Below code doesn't set the `Content-Type` if `_headers` is defined but there is no `Content-Type` key.